### PR TITLE
Updating branding to 3.0.1-servicing

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

File: src/packages.builds [hunk 1]
```diff
@@ -3,9 +3,10 @@
 
   <PropertyGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
     <AdditionalBuildConfigurations>$(AdditionalBuildConfigurations);package-$(ConfigurationGroup)</AdditionalBuildConfigurations>
+    <BuildAllPackages>false</BuildAllPackages>
```
Comment (ViktorHofer) [inline src/packages.builds]: Why is that change necessary?

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 2]
```diff
@@ -21,13 +21,19 @@
       }
     },
     "Microsoft.Bcl.AsyncInterfaces": {
+      "StableVersions": [
```
Comment (ViktorHofer) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: I assume those changes are auto-generated?

Comment (Anipik) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: yes this is auto-generated using https://github.com/dotnet/corefx/blob/da147ece700c2502b91e57d99213feadcfb9c1f0/src/packages.builds#L29 target

File: src/packages.builds (hunk 1 above)
Comment (Anipik) [reply src/packages.builds]: In the servicing releases, we just publishes the packages that we changed in that particular release cycle. I think that should be the reason.
cc @wtgodbe just to verify as I added most of the stuff from https://github.com/wtgodbe/corefx/commit/6b2de6b88c94fad7a844f4d05b0d18090eb32fa5

Comment (wtgodbe) [reply src/packages.builds]: Should add something similar to https://github.com/wtgodbe/corefx/commit/6b2de6b88c94fad7a844f4d05b0d18090eb32fa5#diff-cb73a137838f73872d476fa5146bfedeR18-R27 as well to ensure that we always build the runtime package + the package baseline, and so we can add specific packages to be built for servicing.

**Comment by joperezr** [comment]
@ericstj I have pushed commit 5746369 which will fix the issue we were seeing in the All configurations leg where the wrong harvest package version was restored. PTAL.

**Comment by joperezr** [comment]
FYI, I did verify and the only global property that we were passing in to every pkgproj was `MSBuildRestoreSessionId` so I'm undefining it as part of the change.

**Comment by ericstj** [comment]
Build is failing to test because it has nothing to test.   We need to add a condition here:
https://github.com/dotnet/corefx/blob/eda554cfbcceda02db621ab8233d1b53e653681a/pkg/test/testPackages.proj#L161

Probably `'@(SupportedPackage)' != ''"`

/cc @safern

**Comment by ericstj** [comment]
Looks like the piece that uploads to Helix needs a condition as well.

**Comment by Anipik** [comment]
> Looks like the piece that uploads to Helix needs a condition as well.

We are using Test(CoreTest) target to upload to Helix. i was not able to find any property to disable or any way to override the target to just an empty target. The only other option i havent tried yet is sending a dummy work item to helix. Do u have any ideas ?

CoreTest is being overridden in the helixsdk

**Comment by safern** [comment]
> Do u have any ideas ?

We could introduce a target in `sendtohelix.proj` that actually is the one that calls the `test` target if the `HelixWorkItem` is not empty: https://github.com/dotnet/corefx/blob/release/3.0/eng/sendtohelix.proj#L93

And then instead of calling the `Test` target directly from the command line (https://github.com/dotnet/corefx/blob/release/3.0/eng/pipelines/helix.yml#L17) we could call that target. 

However, we should only allow empty `HelixWorkItem` for the `allConfigurations` leg.

**Comment by mmitche** [comment]
What's the status of this? 3.0 builds are progressing but continuing to flow 3.0.0 stable packages.

**Comment by Anipik** [comment]
> What's the status of this?

I am trying to fix the workflow here. This is fir on my priority list

**Comment by Anipik** [comment]
@ericstj can you review the latest changes.

**Review by ericstj — APPROVED** [review]
